Kconfig.nrf

```diff
@@ -11,9 +11,23 @@ menu "Nordic nRF Connect"
 config BOARD_THINGY91_NRF9160NS
 	bool
 
-# nRF Connect SDK needs a larger default stacks for running certain tests.
+# Handle default to ensure that BT only will disable ENTROPY_CC3XX,
+# while multi protocol does not.
+config ENTROPY_CC3XX
+	default y if ZIGBEE || OPENTHREAD_NRF_SECURITY
+	default n if BT
+
+config ZIGBEE
+	select ENTROPY_CC3XX
+
+config OPENTHREAD_NRF_SECURITY
+	select ENTROPY_CC3XX
+
+# nRF Connect SDK needs a larger default stacks for running certain tests
+# or if ENTROPY_CC3XX is enabled.
```

Comment:
We could consider making this statement more future proof
```suggestion
# nRF Connect SDK needs a larger default stacks for certain configurations.
```

Reply:
Will be considered as a follow-up task once we get this feature in

Right now it is very explicit on what is going on for the sole purpose of being visible in case we rework this at a later time. There are other locations where the default stack size is manipulated in similar type Kconfig defaults (or defconfigs). Stating "certain configurations" makes it look like this is the place that all this activity will happen (which is not true)...